docs/getting-started-kubernetes.rst

```diff
@@ -16,7 +16,1259 @@
+    cni-plugins-ds-kbvnm                            1/1     Running   0          5d
+    cni-plugins-ds-pcllg                            1/1     Running   0          5d
+    kube-multus-ds-5j6ns                            1/1     Running   0          5d
+    kube-multus-ds-mxgvl                            1/1     Running   0          5d
+    mofed-ubuntu20.04-ds-2zzf4                      1/1     Running   0          5d
+    mofed-ubuntu20.04-ds-rfnsw                      1/1     Running   0          5d
+    whereabouts-nw7hn                               1/1     Running   0          5d
+    whereabouts-zvhrv                               1/1     Running   0          5d
+    ...
+
+The ``pod.yaml`` configuration file for such a deployment:
+
+.. code-block:: yaml
+
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: testpod1
+      annotations:
+        k8s.v1.cni.cncf.io/networks: example-sriov-network
+    spec:
+      containers:
+      - name: appcntr1
+        image: <image>
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          capabilities:
+            add: ["IPC_LOCK"]
+        resources:
+          requests:
+            nvidia.com/sriov_resource: '1'
+          limits:
+            nvidia.com/sriov_resource: '1'
+        command:
+        - sh
+        - -c
+        - sleep inf
+
+---------------------------------------------------------------------------
+SR-IOV Network Operator Deployment – Parallel Node Configuration for SR-IOV
+---------------------------------------------------------------------------
+
+.. warning:: This is a Tech Preview feature, which is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
+
+To apply SriovNetworkNodePolicy on several nodes in parallel, specify the ``maxParallelConfiguration`` option in the SriovOperatorConfig CRD:
+
+.. code-block:: bash
+
+    kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "maxParallelNodeConfiguration": 0 } }' --type='merge'
+
+--------------------------------------------------------------------------
+SR-IOV Network Operator Deployment – Parallel NIC Configuration for SR-IOV
+--------------------------------------------------------------------------
+
+.. warning:: This is a Tech Preview feature, which is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
+
+To apply SriovNetworkNodePolicy on several nodes in parallel, specify the ``featureGates`` option in the SriovOperatorConfig CRD:
+
+.. code-block:: bash
+
+    kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "featureGates": { "parallelNicConfig": true  } } }' --type='merge'
+
+---------------------------------------------------------------------------
+SR-IOV Network Operator Deployment – Parallel Node Configuration for SR-IOV
```

Comment:
Should be:
`SR-IOV Network Operator Deployment – SR-IOV Using the systemd Service `